src/gatsby/src/components/park/parkDates.js

```diff
@@ -46,11 +46,7 @@ export const ReservationButtons = ({ campingTypeCode, parkOperation }) => {
       buttons: [
         {
           label: "Book groupsite",
-          fieldName: "frontcountryGroupReservationUrl"
-        },
-        {
-          label: "Book groupsite",
-          fieldName: "backcountryGroupReservationUrl"
+          fieldName: "frontcountryGroupReservationUrl" || "backcountryGroupReservationUrl"
```

Comment:
"frontcountryGroupReservationUrl" is a string so the first half will always be true

Reply:
Doh yes it is just an object value ... 😅